keyboards/sirius/unigo66/keymaps/default/config.h

```diff
@@ -0,0 +1,6 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
```

Comment:
Line 4 should be updated as below. QMK's build script automatically includes all the `config.h` files in the build tree, so this line is redundant. There should be a note, though, for users who copy the keymap to know where the overrides go.

```suggestion
// place overrides here
```